FreeRTOS/FreeRTOS
# Add Kernel Unit Tests Environment

Add Kernel Unit Tests Environment

Description
-----------
Implementing a build environment with a skeleton file to unit test queue.c

Test Steps
-----------
Run
```
$ make coverage
```


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
